app/Scope/ActiveScope.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Scopes;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Scope;
+
+class ActiveScope implements Scope
+{
+    public function apply(Builder $builder, Model $model)
+    {
+        $model::whereNotNull('deleted_at');
```

Comment:
deleted_at not null thì là đã xóa chứ sao active nhỉ?

Reply:
à đúng rồi, em nhầm logic chỗ này